tests/e2e/scale/conftest.py

```diff
@@ -0,0 +1,28 @@
+import logging
+from ocs_ci.framework import config
+
+log = logging.getLogger(__name__)
+
+
+def pytest_collection_modifyitems(items):
+    """
+    A pytest hook to skip certain tests when running on vms.
+
+    Args:
+        items: list of collected tests
+
+    """
+    skip_list = [
+        'test_scale_pvc_expand',
+        'test_ceph_pod_respin_in_scaled_cluster'
+    ]
+    if config.ENV_DATA['platform'].lower() == "vsphere":
```

Comment:
you can use VSPHERE_PLATFORM  from https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L686

Reply:
Changed to use constants.VSPHERE_PLATFORM